tidymodel-prediction-workflows-inside-snowflake-with-orbital/tidymodel-prediction-workflows-inside-snowflake-with-orbital.md

```diff
@@ -0,0 +1,1145 @@
+author: Sara Altman
+id: tidymodel-prediction-workflows-inside-snowflake-with-orbital
+summary: Tidymodel Prediction Workflows Inside Snowflake with Orbital
+categories: Data-Science
+environments: web
+status: Published
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Data Science, R, Posit Workbench, Native Applications 
+
+#  Tidymodel Prediction Workflows Inside Snowflake with Orbital
+
+## Overview
+Duration: 1
+
+### Overview
+
+This Quickstart will demonstrate how to run model predictions inside Snowflake
+using the [orbital R package](https://orbital.tidymodels.org/) and the [Snowflake Native Posit Workbench App](https://app.snowflake.com/marketplace/listing/GZTSZMCB69/posit-pbc-posit-workbench?search=posit+pbc). 
+
+With the orbital package, you can:
+
+1. Speed up model predictions by running them directly in a database like Snowflake.
+2. Easily share your models with others by storing predictions in a Snowflake table or view. 
+
+We'll use loan data from [LendingClub](https://www.lendingclub.com/) to build an 
+example model in this Quickstart. 
+
+### What You'll Need
+
+- Familiarity with R
+- Familiarity with modeling in R with the [tidymodels framework](https://www.tidymodels.org/) 
+- The ability to launch Posit Workbench from [Snowflake Native Applications](https://docs.posit.co/ide/server-pro/integration/snowflake/native-app/). This can be provided by an administrator with the `accountadmin` role.
+- A Snowflake account with an `accountadmin` role or role that allows you to:
+  - Create databases, schemas and tables
+  - Create stages
+  - Load data from S3
```

Comment:
```suggestion
- Familiarity with R
- Familiarity with modeling in R with the [tidymodels framework](https://www.tidymodels.org/) 
- The ability to launch Posit Workbench from [Snowflake Native Applications](https://docs.posit.co/ide/server-pro/integration/snowflake/native-app/). This can be provided by an administrator with the `accountadmin` role.
- A Snowflake account with an `accountadmin` role or role that allows you to:
  - Create databases, schemas and tables
  - Create stages
  - Load data from S3
- Access to [Posit Connect](https://posit.co/products/enterprise/connect/)
```